src/tests/ftest/cart/protocol_info.c

```diff
@@ -0,0 +1,67 @@
+/*
+ * (C) Copyright 2018-2022 Intel Corporation.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause-Patent
+ */
+
+#include <cart/api.h>
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#define NWIDTH 20
+
+static int
+print_info(const char *info_string)
+{
+	struct crt_protocol_info *protocol_infos = NULL, *protocol_info;
+	int                       ret;
+
+	ret = crt_protocol_info_get(info_string, &protocol_infos);
+	if (ret != DER_SUCCESS) {
+		fprintf(stderr, "crt_protocol_info_get() failed (%d)\n", ret);
```

Comment:
use D_ERROR() for these, so that all tests can end up writing to the same log files in CI